msi/GitChooseEnvironmentDlg.wxs

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All things that can be fixed in time, especially since we're on WiX v4 where we can take larger features into the WiX Toolset to support it.